guides/common/modules/con_puma-threads.adoc

```diff
@@ -1,21 +1,13 @@
 [id="Puma_Threads_{context}"]
 = Puma threads
 
-Number of Puma threads (per Puma worker) is configured by using two values: `threads_min` and `threads_max`.
+Number of Puma threads (per Puma worker) is configured using two values: `threads_min` and `threads_max`.
```

Comment:
```suggestion
Number of Puma threads (per Puma worker) is configured by using two values: `threads_min` and `threads_max`.
```

This was changed recently by Lena in daa5114e654bc96cb1840c7ed066e7940372adec

Reply:
Actually, just to clarify, "by" should not be needed in your suggestion case because it's preceded by a verb: there is no risk of ambiguity in the sense of https://github.com/theforeman/foreman-documentation/issues/3141#issuecomment-2232701827. When there is no risk of ambiguity, it's actually preferable to stick to "using" (without "by") because it's easier to understand and shorter. (BTW, it's not just the verb "using" that can be problematic. So if we required "by using" on each occurrence, we would need to apply the same requirement to other verbs and that would be... weird :))

If anything, I recommend to use active voice:

```suggestion
You can configure the number of Puma threads (per Puma worker) by using these values: `threads_min` and `threads_max`.
```

... in which case we _do_ need "by using" because otherwise, this could be interpreted as either "configure the number of threads that use" or "configure the number of threads by using" :upside_down_face: 

Or, even more simply:

```suggestion
To configure the number of Puma threads (per Puma worker), use these values: `threads_min` and `threads_max`.
```